src/pages/home/Home.tsx

```diff
@@ -19,7 +19,9 @@ const Home: FC<HomeProps> = () => {
           This is an example of how you can create a web page. This example
```

Comment:
remove this sample text and add some brief text welcoming CFL contributor's our contributor service and encouraging them to become a contributor.